### PR TITLE
feature/AT-RSC-UI-AddStory button

### PR DESCRIPTION
**Pull request by MajoYanez**
Added a button for story in index,a folder for story in views/admin and the file add-story.hbs with the code of the form to add story

@fasaga

### CODE REVIEW

File: src/views/admin/index.hbs [hunk 1]
```diff
@@ -47,6 +47,11 @@
                     <a class="btn btn-primary" href="/admin/metrics">&raquo; Metrics</a>
                 </td>
             </tr>
+            <tr>
```
Comment (at-internship) [inline src/views/admin/index.hbs:50]: should be
```
<tr>
                <td style="text-align: left;">
                    <a class="btn btn-primary" href="/admin/story">&raquo; Story</a>
                </td>
            </tr>
```

**Review by at-internship — CHANGES_REQUESTED** [review]
these routes are needed

// AT-RESOURCES - Render Story
router.get("/story", story);

// AT-RESOURCES - Render AddStory
router.get("/story/add", renderAddStoryForm);

// AT-RESOURCES - AddStory
router.post("/story/add", addStory);

**Comment by fasaga** [comment]
@at-internship Routes are not part of her task, they will be added by another team member as well as the controller methods, thanks

**Review by at-internship — APPROVED** [review]
Approved

**Review by fasaga — APPROVED** [review]

**Review by raaul-at — APPROVED** [review]

**Pull request merged**